scripts/dev/edit_cluster_config.sh

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env bash
+
```

Comment:
Would be good to describe what it does here and even create some md file.

Reply:
Done. Is it OK?